src/handlers/device.js

```diff
@@ -3,8 +3,7 @@ import util from 'util';
 import { sleep } from '../util';
 
 let debug = Debug('testdroid-proxy:handler:device');
-let buildLabelGroupName = 'Build Identifier';
-let flashProjectName = 'flash-fxos';
+let flashProjectName = 'flash-fxos-new-url';
```

Comment:
Is there a url or some info you can attach to this to describe how to configure it maybe?